submit.py

```
@@ -0,0 +1,53 @@
+import re
+import json
+import subprocess
+
+# 1. Add support for epics
+# 2. Add support for subtasks
+# 3. Add support for assigning issues to users
+
+directory = "tmp"
+with open(f"{directory}/metadata.json", "r") as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)